styles/AdvancedEditor.css

```diff
@@ -626,7 +626,9 @@ span.comment-marker {
 .keywords .keyword-group {
   display: inline;
 }
-
+.keyword-group .keyword {
+  border: 1px solid #C9C9C9;
+}
```

Comment:
no need for that

Reply:
why ? this is requested by the new design , to add a border for keywords 

![image](https://github.com/user-attachments/assets/45572f11-6200-4bc4-82da-69db114c2391)

Reply:
so the ticket is a bug with style improvement?

Reply:
exactly , 
![image](https://github.com/user-attachments/assets/2b5f6142-29b6-453b-a603-43eaeb70d961)

![image](https://github.com/user-attachments/assets/20bb37ed-1f58-421a-bba6-90ed7edd82b3)


my bad ! I forgot to add a description to the PR